### PR TITLE
Strezless Musick Productionz

### LINKED ISSUE #12: Strezless Productionz Team
Experimenting a music career take off from scratch
any one with new ideas welcomed and will definitely be taken into consideration
Starring Artist known as Sin Estres 
Omar Orrantia

<iframe src="https://archive.org/embed/not-another-broken-heart" width="500" height="140" frameborder="0" webkitallowfullscreen="true" mozallowfullscreen="true" allowfullscreen></iframe

### LINKED ISSUE #52: La matematicas de la Vida simple
no es desprecio es aprecio.

el amor = verdad

la mentira = es odio

el odio = negativo

el amor = positivo

lo que cosechas ÷ lo que siembras

te dejo pa que resuelvas la matematicas 

Cristo es la verdad

### LINKED ISSUE #55: Strezless Productionz Team
Experimenting a music career take off from scratch
any one with new ideas welcomed and will definitely be taken into consideration
Starring Artist known as Sin Estres 
Omar Orrantia

### PR DESCRIPTION
**Pull request by SinEstres**
Sin Estres Orrantia
Artist
Music Distribution Label in the making
Training
Founder
Rap
Hip Hop

### CODE REVIEW

**Comment by SinEstres** [comment]
[Strezless Musick Productionz Domain](theofficialsinestres.com)

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    # Runs a single command using the runners shell
+    - name: Run a one-line script
+      run: echo Hello, world!
+
+    # Runs a set of commands using the runners shell
+    - name: Run a multi-line script
+      run: |
+        echo Add other actions to build,
+        echo test, and deploy your project.
```
Comment (SinEstres) [inline .github/workflows/blank.yml:33]: Strezless Musick Productionz

**Review by SinEstres — COMMENTED** [review]
[The homies.pdf](https://github.com/SinEstres/Strezless-Musick-Productionz-Founder/files/5175028/The.homies.pdf)

**Review by SinEstresOrrantia — APPROVED** [review]
Strezless Musick Productionz by Sin Estres approved of whatever this is for

**Review by StrezlessMusick — COMMENTED** [review]
Review

File: .github/workflows/blank.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
```
Comment (Strezless-Musick-Metadata) [inline .github/workflows/blank.yml:5]:
```suggestion
# 030eb74
```

**Review by Strezless-Musick-Metadata — APPROVED** [review]

File: .github/workflows/blank.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    # Runs a single command using the runners shell
+    - name: Run a one-line script
+      run: echo Hello, world!
+
+    # Runs a set of commands using the runners shell
+    - name: Run a multi-line script
+      run: |
```
Comment (Strezless-Musick-Metadata) [inline .github/workflows/blank.yml:31]:
```suggestion
      run: | Python for Strezless Musick By Sin Estres
```


```suggestion
      run: | tagging
```

File: .github/workflows/blank.yml (hunk 1 above)
Comment (Strezless-Musick-Metadata) [inline .github/workflows/blank.yml:33]: echo **Strezless Musick Productionz**

**Review by Strezless-Musick-Metadata — APPROVED** [review]

File: .github/workflows/blank.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
```
Comment (Strezless-Musick-Metadata) [inline .github/workflows/blank.yml:6]:
```suggestion
# Strezless Musick Meta
```

@Strezless-Musick-Metadata 
@SinEstres @StrezlessMusick @

File: .github/workflows/blank.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+on:
```
Comment (Strezless-Musick-Metadata) [inline .github/workflows/blank.yml:7]: <**Omar Dejesu Orrantia**=**Sin Estres**)<Merge with **Strezless Musick** _Metadata_>  _https://sinestres.substack.com_
# @Strezless-Musick-Metadata

File: .github/workflows/blank.yml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  push:
```
Comment (Strezless-Musick-Metadata) [inline .github/workflows/blank.yml:8]: #@SinEstres @Strezless-Musick-Metadata @StrezlessMusick

File: .github/workflows/blank.yml [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
```
Comment (Strezless-Musick-Metadata) [inline .github/workflows/blank.yml:14]: **Songs as Work on BMI
- -  ~~~~**

Comment (Strezless-Musick-Metadata) [reply .github/workflows/blank.yml:14]: Public Figure Merge **Sin Estres** Public Info https://g.co/kgs/5NU6Rs
> 
- https://music.apple.com/us/artist/sin-estres/1500746854
-

File: .github/workflows/blank.yml (hunk 1 above)
Comment (StrezlessMusick) [reply .github/workflows/blank.yml:33]:
```
<rss version="2.0">
<channel>
<language>en</language>
<title>Recent activity by Strezless</title>
<pubDate/>
<lastBuildDate/>
</channel>
</rss>
```

**Review by Sin-Estres-Orrantia — COMMENTED** [review]
[CI](https://github.com/SinEstres/Strezless-Musick-Productionz-Founder/actions/workflows/blank.yml)

**Pull request merged**